1044 add footer for all pages

minump opened the pull request:
## Description
Footer is added to all pages
Footer div is moved from home page to topnav page. the topnav page is included in all pages.
Some style changes are done.

**Fixes #1044**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

sandeep-ps commented on contributions/catalog/webapps/templates/topbottomnav.html:
```diff
@@ -30,4 +30,9 @@ <h1 class="catalog-home">Contributions Catalog</h1>
 </div>
 
+<div class="rokwire-footer">
+    <img src= "{{ url_for('static', filename='powered-by-rokwire-no-i-block.png') }}" alt="footer logo" />
```
I think we may have to use all images mentioned in the srcset attribute in the design. The current rendering looks slightly blurred. 
<img width="202" alt="image" src="https://user-images.githubusercontent.com/4306604/216075666-43d0778a-60c3-450d-ba52-af8095fc4425.png">

minump replied:
updated

sandeep-ps replied:
Thanks.

sandeep-ps commented on contributions/catalog/webapps/templates/contribute/talent_details.html:
```diff
@@ -20,7 +20,7 @@
 <body onload=start()>
 <div class="header">
-    {% include 'topnav.html' %}
+    {% include 'topbottomnav.html' %}
```
I would also suggest using a better readable file name like top_bottom_nav.html. Thanks.

minump replied:
updated

sandeep-ps replied:
Thanks.

minump commented:
Updated with feedback. Ready for review

sandeep-ps commented:
Thanks for the changes. Approved.

minump commented:
Implement footer in another page , say footer.html (separate from topnav.html). Include footer.html in all pages, at the bottom of all content.
By this, we can remove the rokwire-footer margin:437px styling, keep the position of the footer below all contents. Adjust the main container margin and padding accordingly.

minump commented:
Converting to draft to work on changes.

minump commented:
> Implement footer in another page , say footer.html (separate from topnav.html). Include footer.html in all pages, at the bottom of all content. By this, we can remove the rokwire-footer margin:437px styling, keep the position of the footer below all contents. Adjust the main container margin and padding accordingly.

Added a footer.html page. Removed margin for footer class. Placed footer.html at the bottom of all pages.

minump commented:
@sandeep-ps Detect-secrets are failing. I cross-checked and dont think I pushed any secrets.
@sandeep-ps and @bingzhang Ready for review.

sandeep-ps commented:
Hi, Minu, I have restarted reviewing this. I'm getting content blocked by the footer. Should we see this after your new updates? Could you please look into this?

<img width="1291" alt="image" src="https://user-images.githubusercontent.com/4306604/217930857-48c21673-90de-4324-9da5-210260041f23.png">

minump commented:
Footer image with padding: 30px 412px 0px 412px;
![Screen Shot 2023-02-09 at 3 42 15 PM](https://user-images.githubusercontent.com/22112888/217946031-b976ba0e-318d-48e6-9c92-3f5f0e46405f.png)

Footer image with padding: 85px 363px 26.8px;   (as per zeplin design)
![Screen Shot 2023-02-09 at 3 41 48 PM](https://user-images.githubusercontent.com/22112888/217946111-fac8c372-b1fe-48a6-9e8b-a2bf21809fdb.png)

The footer is bigger as per zeplin design. This is currently implemented in the PR.

minump commented:
@sandeep-ps updated footer styling. ready for review.

minump commented:
@sandeep-ps styling and footer updated.. I did check all pages and in incognito.. Hopefully I caught all errors/position misplacements. Let me know if you find something amiss.

@sandeep-ps @bingzhang Ready for review.

sandeep-ps commented:
The changes look good to me. Thanks.

I saw another thing while reviewing this. Both on edit and create pages, there is some blank space on both the left and right sides. Please create a follow-up issue to fix this, which can be planned for the next sprint. Thanks.

<img width="266" alt="image" src="https://user-images.githubusercontent.com/4306604/218181854-9bd2291e-e09e-480a-97c9-fc750cd49fde.png">

minump commented:
> The changes look good to me. Thanks.
> 
> I saw another thing while reviewing this. Both on edit and create pages, there is some blank space on both the left and right sides. Please create a follow-up issue to fix this, which can be planned for the next sprint. Thanks.
> 
> <img alt="image" width="266" src="https://user-images.githubusercontent.com/4306604/218181854-9bd2291e-e09e-480a-97c9-fc750cd49fde.png">

Issue created for further UI change in footer. https://github.com/rokwire/rokwire-building-blocks-api/issues/1081

minump commented:
Merging to develop. Bypassing brach protections.
